test/unit/gateways/stripe_test.rb

```diff
@@ -342,8 +342,7 @@ def test_successful_refund
     assert response = @gateway.refund(@refund_amount, 'ch_test_charge')
     assert_success response
 
-    assert_equal 'ch_test_charge', response.authorization
-    assert response.test?
```

Comment:
Oh ok - TIL. I guess it just uses the associated charge_id to figure it out. This is fine, then